yellow-paper/docs/logs/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Logs
+---
+
+Logs on Aztec are similar to logs on Ethereum and their goal is to allow smart contracts to communicate arbitrary data to the outside world.
+Logs are events which are emitted during contract function execution.
+Aztec protocol gives users the following assurances:
+1. The logs get published on the chosen data availability layer,
```

Comment:
Was this with ability to publish multiple places? 👀 Might be better to just say it is published since we don't know about the rest really.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3582/commits/7c1242a272fe3de22c35d090d640d77393643dfc